implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java

```diff
@@ -1,16 +1,14 @@
 package org.codemc.worldguardwrapper.implementation.v7;
 
+import com.sk89q.worldedit.BlockVector;
+import com.sk89q.worldedit.BlockVector2D;
```

Comment:
As the sk89q repository has been fixed, the BlockVector and BlockVector2D don't exist anymore in the latest WorldEdit snapshot and have been replaced with BlockVector2 and BlockVector3. So, does this actually build?

Reply:
I'll try to force a dependency update in my dev environment :/